go-ssb-bindings/debug.go

```diff
@@ -107,6 +109,19 @@ func ssbRepoStats() *C.char {
 	counts.Messages = rootSeq.Seq()
 	counts.Messages += 1 // 0-indexed (empty is -1)
 
+	lm, err := sbot.RootLog.Get(rootSeq)
+	if err != nil {
+		retErr = errors.Wrap(err, "RepoStats: could not get the last message hash")
+		return nil
```

Comment:
I think we shouldn't return nil here as we may have a valid feed count and/or message count. For instance, when we onboard a new user, feeds and messages are empty but it doesn't have a last message hash. This is way the unit tests are failing.

Reply:
Fixed ^w^